raven/contrib/tornado/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 from tornado import ioloop
 from tornado.httpclient import AsyncHTTPClient, HTTPError
+from tornado import web
```

Comment:
Yeah, that's fine. I understand the confusion. :) I had to double check the source to make sure they were indeed two separate objects at different paths and not an alias for each other.